compiler/optimizer/UseDefInfo.hpp

```diff
@@ -114,6 +114,10 @@ class TR_UseDefInfo
       // BitVector for temporary work. Used in buildUseDefs.
       TR_BitVector _workBitVector;
 
+      // Cache results from isTrivialNode
+      TR_BitVector _doneTrivialNode;
```

Comment:
don't these need to be initialized with the correct region?

Reply:
The missing initialization in aux's region has been added.